modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -79,7 +79,7 @@ public BatchEnginePortletDataHandler(
 		String className, CompanyLocalService companyLocalService,
 		ExportImportVulcanBatchEngineTaskItemDelegate
 			exportImportVulcanBatchEngineTaskItemDelegate,
-		String itemClassName, String taskItemDelegateName,
```

Comment:
@4lejandrito the change to BatchEnginePortletDataHandler is logically wrong. Its base class BasePortletDataHandler already has a _portletId field with getter and setter. With this change, assume you have a BatchEnginePortletDataHandler instance, and you call setPortletId() on it, it modifies the BasePortletDataHandler's _portletId field. But then you call getPortletId() on this BatchEnginePortletDataHandler instance you will see it still returns unchanged value. 
What's reason for this change? If you need to assign a portletId to it, you can just call setPortletId on the instance, no need to change this BatchEnginePortletDataHandler class for it.